### DISCUSSION TITLE
IPad Termius missing cursor

### DISCUSSION THREAD

**Discussion by danriedl** [helix-editor/helix#8981] [Troubleshooting]
### Summary

This is a very weird bug Report but i tried helix over ssh on termius on my ipad.
Somehow i cant see the cursor but on vi it works.
Any hints of how to get it working?

Thanks!

https://github.com/helix-editor/helix/assets/107212962/35a56c38-0e29-474d-bf33-773678fac771

### Reproduction Steps

 

I tried this:

1. `hx` in a ssh session with termius app on ipad

I expected this to happen:
Navigation is working as expected

Instead, this happened:
Cursor not showing up

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Ubuntu

### Terminal Emulator

Termius

### Installation Method

Release

### Helix Version

Helix 23.10 (f6021dd0)

**Comment by eli-yip** [comment]
Same bug on termius 9.1 amd helix 24.03/24.07

**Reply by HW-Yue** [reply]
so how to deal with it
